client/jetpack-connect/auth-logged-in-form.jsx

```diff
@@ -66,7 +68,10 @@ class LoggedInForm extends Component {
 	static propTypes = {
 		isSSO: PropTypes.bool,
 		isWoo: PropTypes.bool,
-		jetpackConnectAuthorize: PropTypes.shape( {
+
+		// Connected props
+		authAttempts: PropTypes.number.isRequired,
+		authorizationData: PropTypes.shape( {
```

Comment:
Diff is a but weird, but `jetpackConnectAuthorize` is renamed to `authorizationData` and moved with other connected props. `user` is moved to connected props.